common/predictive-text/src/web/worker.ts

```diff
@@ -1,5 +1,5 @@
 import unwrap from '../unwrap.js';
-import { LMLayerWorkerCode, LMLayerWorkerSourcemapComment } from "@keymanapp/lm-worker/build/lib/worker-main.wrapped.js";
+import { LMLayerWorkerCode, LMLayerWorkerSourcemapComment } from "@keymanapp/lm-worker/build/lib/worker-main.wrapped.min.js";
```

Comment:
This feels like a linker detail in the import, should be resolved in build process rather than in source?

Reply:
`esbuild` can't handle or resolve dynamic imports when importing.  It's pretty much one or the other, hence the parallel worker.ts and sourcemappedWorker.ts in both the `node` and `web` folders of `common/predictive-text`.  I'd just messed up the link for this file on the prior PRs.